packages/client/hmi-client/src/components/workflow/tera-simulate-julia.vue

```diff
@@ -43,12 +43,28 @@
 			/>
 			<Button
 				class="add-chart"
-				text
-				:outlined="true"
-				@click="saveDataset(projectId, completedRunId)"
-				label="Save as Dataset"
-				icon="pi pi-save"
-			/>
+				title="Saves the current version of the model as a new Terarium asset"
+				@click="showSaveInput = !showSaveInput"
+			>
+				<span class="pi pi-save p-button-icon p-button-icon-left"></span>
+				<span class="p-button-text">Save as</span>
+			</Button>
+			<span v-if="showSaveInput" style="padding-left: 1em; padding-right: 2em">
```

Comment:
if we can classes/ids over inline styles

Reply:
Noted. I've lifted this code from the chatty component so once we have that it can take it too.